examples/Async.java

```diff
@@ -5,58 +5,60 @@
 // 10 minutes. This is useful when creating many documents in parallel, or very
 // large documents with lots of assets.
 //
-// DocRaptor supports many options for output customization, the full list is
-// https://docraptor.com/documentation/api#api_general
+// DocRaptor supports many CSS and API options for output customization. Visit
+// https://docraptor.com/documentation/ for full details.
 //
 // You can run this example with: ./script/run_java_file examples/Async.java
 
+import com.docraptor.*;
 import java.io.*;
 import java.net.*;
-import com.docraptor.*;
+import java.nio.file.*;
+
+public class Sync {
```

Comment:
Seems like this class name should roughly match the filename. Perhaps `AsyncTest` in order to indicate it's a test as well? I'm not super sure of the conventions for test class names in java.

Reply:
What if it was just `DocRaptor`? And it's not a test, it is just tested to ensure it works before publishing.

Reply:
Ah, yeah, that makes sense. My brain had conflated the tests and the examples in these repos. Maybe we can DRY up the tests at some point as well, but that can be later. 

Yeah, `DocRaptor` seems okay, maybe `DocRaptorExample` to differentiate it from the official DocRaptor library classes?

Reply:
`DocRaptorExample` sounds good. Thanks!

Reply:
Fixed